SVG/stroke-dasharray/pieChart.xsl

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="3.0"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    exclude-result-prefixes="xs math">
+    
+    <xsl:output method="xml" indent="yes"/> 
+    
+    <xsl:variable name="KG" select="collection('xml/?select=*.xml')"/>
```

Comment:
@Yuying-Jin If you're simplifying the XML to just one file for the purposes of this tutorial, you will not need the $KG `collection()` variable here (so you can just remove this line).